app/config/Seeds/ArticlesSeed.php

```diff
@@ -0,0 +1,61 @@
+<?php
+declare(strict_types=1);
+
+use Cake\Log\Log;
+use Phinx\Seed\AbstractSeed;
+
+/**
+ * Articles seed.
+ */
+class ArticlesSeed extends AbstractSeed
+{ 
+    public function getDependencies(): array
+    {
+        return [
+            'UsersSeed',
+        ];
+    }
+
+    /**
+     * Run Method.
+     *
+     * Write your database seeder using this method.
+     *
+     * More information on writing seeds is available here:
+     * https://book.cakephp.org/phinx/0/en/seeding.html
+     *
+     * @return void
+     */
+    public function run(): void
+    {
+        try {
+            // Define the data to insert
+            $title = 'First Post';
+            $articlesData = [
+                [
+                    'user_id' => 1,
+                    'title' => $title,
+                    'slug' => 'first-post',
+                    'body' => 'This is the first post.',
+                    'published' => 1,
+                    'created' => date('Y-m-d H:i:s'),
+                    'modified' => date('Y-m-d H:i:s'),
```

Comment:
In the future, for the date/time related function, please use the CakePHP FrozenDate/FrozenTime instead of PHP date function. Refer https://book.cakephp.org/4/en/core-libraries/time.html

The current LOC is OK for now.